topic_5/.gitignore

```diff
@@ -0,0 +1,11 @@
+topic_5/.project
+topic_5/.classpath
+topic_5/target/classes/com/globant/dao/MongoConnection.class
```

Comment:
You can ignore a directory, in this case /target instead of ignoring files one by one